tox.ini

```
@@ -11,49 +11,40 @@
 # then then deleting compiled files has been found to fix it: `find . -name \*.pyc -delete`
 
 [tox]
-envlist = py37-sphinx3
+envlist = py38-sphinx3
```

```suggestion
envlist = py39-sphinx3
```

Perhaps we should stick with the latest supported Python version as the default tox environment?

---

Sounds good